app/models/phase.rb

```diff
@@ -11,43 +11,62 @@ class Phase < ActiveRecord::Base
   ##
   # Associations
   belongs_to :template
-  has_many :sections, -> { order(:number => :asc) }, dependent: :destroy
+
+  has_one :prefix_section, -> (phase) {
```

Comment:
How do the prefix and suffix sections work when there are no sections that arent modifiable?

Reply:
`prefix_section` will return nil, since it's a *has one* association.
`suffix_sections` will return an *ActiveRecord::Relation* object in any case